packages/pg-cluster/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@databases/pg-cluster",
+  "version": "0.0.0",
+  "description": "",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "dependencies": {},
+  "devDependencies": {
+    "@databases/pg": "^0.0.0",
+    "@databases/sql": "^0.0.0"
+  },
+  "peerDependencies": {
+    "@databases/pg": "*",
+    "@databases/sql": "*"
+  },
```

Because `@databases/pg` re-exports bot `sql` and the `SQLQuery` type, you can reduce the dependencies to only `@databases/pg`:

```suggestion
  "devDependencies": {
    "@databases/pg": "^0.0.0"
  },
  "peerDependencies": {
    "@databases/pg": "*"
  },
```